rfcs/0102-moderation-team.md

```diff
@@ -0,0 +1,94 @@
+that requires moderation, and we'd like to help the community become more
+self-regulating.
+
+(adopted from #98)
+
+# Detailed design
+[design]: #detailed-design
+
+The Moderation Team is a volunteer group that may receive, invite, and evaluate
+applications to the team or alter the composition at any time with approval
+from the NixOS Foundation board or by any individual choosing to resign. The
+team's composition, contact information, and announcements should be maintained
+at
+[https://nixos.org/community/teams/moderation.html](https://nixos.org/community/teams/moderation.html).
+The team shall perform moderation activities on behalf of the NixOS Foundation
+for the discussions in the [official project
+spaces](https://nixos.org/community/index.html) as well as unofficial spaces
+that seek and reach such an agreement with the team. The team should utilize
+the [NixOS Foundation mission](https://nixos.org/community/index.html) and the
+following statement during their duties:
+
+```
+The NixOS Foundation aims to promote participation without regard to gender,
+sexual orientation, disability, ethnicity, age, or similar personal
+characteristics. We want to strive to create and foster community by providing
+an intentionally welcoming and safe environment where all feel valued and cared
+for, and where all are given opportunity to participate meaningfully. The
+Foundation will work with the community in service of this goal.
+```
+
+ref: [twitter](https://twitter.com/grhmc/status/1390775249424338944)
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+The initial Moderation Team is defined to be @grahamc, @zimbatm, @domenkozar,
+and @ryantm.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+* The moderation team has limited guidance from this RFC on the processes and
+  procedures of the team.
+* This RFC is designed to address a narrow part of a current issue facing the
+  community. Additional RFCs may be needed to address additional concerns.
+* As this is a controversial topic there is a potential this RFC does not have
+  enough detail to be acceptable by the overall community.
+
+# Alternatives
+[alternatives]: #alternatives
+
+* An existing [RFC 98][].
+* Do nothing.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+* Does the team require additional guidance?
+* Does the NixOS Foundation board want to be involved in this manner?
+
+# Future work
+[future]: #future-work
+
+* A potential RFC providing additional guidance and detail for the moderation
+  team's activities and functions.
+* The role of the moderation team could evolve through an effort similar to
+  [RFC 98][] into taking a broader community leadership responsibility as a
+  'Leadership Team' or 'Community Team'.
+* Work on clarifying community norm guidelines. This can include adopting
+  typical governance tools, such as [Contributor
+  Covenant](https://www.contributor-covenant.org/), Statements of Values, and
```

Comment:
```suggestion
  typical governance tools, such as Contributor
  Covenant, Statements of Values, and
```

Reply:
@7c6f434c removing the link takes out a bit of its accent. Would that be ok?

Reply:
I guess if we are speaking of typical tools, one could also add a generic mention of Code of Conduct (and then indeed it would read like a uniform enumeration of kinds of documents often taken)

Reply:
Perfect, that was the intention. Our (my) concrete impressions during redaction might have skewed that intent a bit.

Thank you!